addons/transitions/Transitions.gd

```diff
@@ -82,12 +82,12 @@ func _common_pre_fade(fade_type, fade_time_seconds:float, shader_image:StreamTex
 	# game will look fine, though. To fix this, scale as needed.
 	# eg. if the game is 960x540 but test_width/test_height is 1600x900, the
 	# screenshot is 1600x900; so it looks zoomed in :facepalm:
-	var game_width:int = ProjectSettings.get_setting("display/window/size/width")
-	var game_height:int = ProjectSettings.get_setting("display/window/size/height")
+	var game_width:float = ProjectSettings.get("display/window/size/width")
```

Comment:
According to [the docs](https://docs.godotengine.org/en/stable/classes/class_projectsettings.html), these are currently `int` values. I also don't see `get` mentioned in the docs, but I do see `get_setting`. What version of Godot are you using?

Reply:
that is meant to avoid casting them to float afterwards for float division, I did it for code cleaning.

Reply:
Would it make more sense to create a separate `float` variable for that instead? e.g. `var some_width:float = game_width / 1.0`